src/app/scripts/cac/control/cac-control-sidebar-directions.js

```diff
@@ -253,6 +261,21 @@ CAC.Control.SidebarDirections = (function ($, Control, BikeModeOptions, Geocoder
         }
     }
 
+    /**
+     * Handles hover events to highlight a given itinerary
+     */
+    function onItineraryHover(event, itinerary) {
+        if (itinerary) {
+            $(options.selectors.itineraryById + currentItinerary.id + '"]')
```

Comment:
Me either.  Went with this to reduce the amount of things to join.  Will go with something else.